File: Swift/프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.swift
import Foundation

func solution(_ babbling:[String]) -> Int {
    let possible = ["aya", "ye", "woo", "ma"]
    
    return babbling.filter { word in
        var temp = word
                            
        for sound in possible {
            temp = temp.replacingOccurrences(of: sound, with: " ")
        }
                            
        return temp.trimmingCharacters(in: .whitespaces).isEmpty
    }.count
}

File: Python3/프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    possible = ["aya", "ye", "woo", "ma"]
    
    for word in babbling:
        temp = word
        
        for sound in possible:
            if sound in temp:
                temp = temp.replace(sound, " ")
        
        if temp.strip() == "":
            answer += 1
    
    return answer